bitnami/jaeger/values.yaml

```diff
@@ -0,0 +1,1117 @@
+  ## Jaeger query.service parameters
+  ##
+  service:
+    ## @param query.service.type Jaeger service type
+    ##
+    type: ClusterIP
+    ## Ports to expose
+    ## ref: https://kubernetes.io/docs/concepts/services-networking/service/#type-nodeport
+    ports:
+      ## @param query.service.ports.api Port for API
+      ##
+      api: 16686
+      ## @param query.service.ports.admin Port for admin
+      ##
+      admin: 16687
+    ## Node ports to expose
+    ## ref: https://kubernetes.io/docs/concepts/services-networking/service/#type-nodeport
+    nodePorts:
+      ## @param query.service.nodePorts.api Node port for API
+      ##
+      api: ""
+      ## @param query.service.nodePorts.admin Node port for admin
+      ##
+      admin: ""
+    ## @param query.service.extraPorts Extra ports to expose in the service (normally used with the `sidecar` value)
+    ##
+    extraPorts: []
+    ## @param query.service.loadBalancerIP LoadBalancerIP if service type is `LoadBalancer`
+    ## ref: https://kubernetes.io/docs/concepts/services-networking/service/#internal-load-balancer
+    ##
+    loadBalancerIP: ""
+    ## @param query.service.loadBalancerSourceRanges Service Load Balancer sources
+    ## ref: https://kubernetes.io/docs/tasks/access-application-cluster/configure-cloud-provider-firewall/#restrict-access-for-loadbalancer-service
+    ## e.g:
+    ## loadBalancerSourceRanges:
+    ##   - 10.10.10.0/24
+    ##
+    loadBalancerSourceRanges: []
+    ## @param query.service.clusterIP Service Cluster IP
+    ## e.g.:
+    ## clusterIP: None
+    ##
+    clusterIP: ""
+    ## @param query.service.externalTrafficPolicy Service external traffic policy
+    ## ref https://kubernetes.io/docs/tasks/access-application-cluster/create-external-load-balancer/#preserving-the-client-source-ip
+    ##
+    externalTrafficPolicy: Cluster
+    ## @param query.service.annotations Provide any additional annotations which may be required.
+    ## This can be used to set the LoadBalancer service type to internal only.
+    ## ref: https://kubernetes.io/docs/concepts/services-networking/service/#internal-load-balancer
+    ##
+    annotations: {}
+    ## @param query.service.sessionAffinity Session Affinity for Kubernetes service, can be "None" or "ClientIP"
+    ## If "ClientIP", consecutive client requests will be directed to the same Pod
+    ## ref: https://kubernetes.io/docs/concepts/services-networking/service/#virtual-ips-and-service-proxies
+    ##
+    sessionAffinity: None
+    ## @param query.service.sessionAffinityConfig Additional settings for the sessionAffinity
+    ## sessionAffinityConfig:
+    ##   clientIP:
+    ##     timeoutSeconds: 300
+    ##
+    sessionAffinityConfig: {}
+  ## Jaeger Query serviceAccount parameters
+  ##
+  serviceAccount:
+    ## @param query.serviceAccount.create Enables ServiceAccount
+    ##
+    create: true
+    ## @param query.serviceAccount.name ServiceAccount name
+    ##
+    name: jaeger-query-sa
```

Comment:
In the Bitnami standard this field is set to "". Right now, if you install a second instance of the chart in the same namespace it will fail because of the colliding service account names

Reply:
Done [c0ca957](https://github.com/bitnami/charts/pull/13480/commits/c0ca9571a089cf22287f768699d7357d7aa6bb98)